features/in_onderzoek.feature

```diff
@@ -69,7 +92,7 @@ Functionaliteit: in onderzoek
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | ouders,partners                 |
```

Comment:
interessant om aan fields toe te voegen bijv. gemeenteVanInschrijving en indicatieCurateleRegister, beide in de persoon resource maar niet afkomstig van de persoon categorie